test/integration/targets/lambda_info/files/mini_lambda.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
```

Comment:
```suggestion
from __future__ import absolute_import, division, print_function
__metaclass__ = type
```
And then the sanity/ignores.txt entry won't be required. We prefer not to put any new items in the ignores.